asv/environment.py

```diff
@@ -916,24 +917,14 @@ def _build_project(self, repo, commit_hash, build_dir):
         """
         Run build commands
         """
-        build_dir_path = Path(build_dir)
-
-        def has_file(file_name):
-            return (build_dir_path / file_name).exists()
 
         cmd = self._build_command
 
         if cmd is None:
-            if has_file('pyproject.toml'):
-                cmd = [
-                    "python -m build",
-                    "python -mpip wheel -w {build_cache_dir} {build_dir}"
-                ]
-            else:
-                cmd = [
-                    "python setup.py build",
-                    "python -mpip wheel -w {build_cache_dir} {build_dir}"
-                ]
+            cmd = [
+                "PIP_NO_BUILD_ISOLATION=0 python -m build",
+                "python -mpip wheel -w {build_cache_dir} {build_dir}"
+            ]
```

Comment:
Why does this run `wheel` after running `build`, which would have already built a wheel?

Also, the call to `build` should probably pass `--wheel`, because I don't think the sdist is used for anything.

Reply:
Oh, I just remembered that `pip wheel` downloads wheels for dependencies as well, so I guess that's why it's used?

The problem seems to be that it says it's building the wheel again, which seems redundant (though it's not verbose, so I can't tell if it's just saying that or is really doing a whole build again.)